### ISSUE TITLE
sprinkle [KMG]i{Bit,Byte} over the logs

### ISSUE THREAD

**Issue by cagney** [libreswan/libreswan#867]
For instance:
```
Sep 21 12:32:48 [tuomo.foobar.fi](http://tuomo.foobar.fi/) pluto[193761]: "foo-v6" [#192](https://github.com/libreswan/libreswan/issues/192): ESP traffic information: in=3GiB out=3GiB
```
it's probably bytes, but everyone talks about giabit ethernet.

**Comment by letoams** [comment]
On Sep 21, 2022, at 08:08, cagney ***@***.***> wrote:
> 
> ﻿
> For instance:
> 
> Sep 21 12:32:48 [tuomo.foobar.fi](http://tuomo.foobar.fi/) pluto[193761]: "foo-v6" [#192](https://github.com/libreswan/libreswan/issues/192): ESP traffic information: in=3GiB out=3GiB
> it's probably bytes, but everyone talks about giabit ethernet.
> 
Is that the case for amounts (and not speeds l) ? So we need another convention type ?

Paul

**Comment by cagney** [comment]
> Is that the case for amounts (and not speeds l) ? So we need another convention type ?

Yes, I'm conflating bits (which crypto and hardware marketing like) with bytes.  In some messages we do provide a gentle nudge in the right direction.  For instance maxBytes=3GiB, which could be written as max=3GiBytes.

**Comment by bleve** [comment]
GiB is gigabytes. That's clear. Uppercase B is byte, lowercase b bit.

**Comment by cagney** [comment]
Wikipedia has:
GiB https://en.wikipedia.org/wiki/Byte#Multiple-byte_units
Gibit https://en.wikipedia.org/wiki/Bit#Unit_and_symbol
So SI specified B for byte, but not b for bit; IEC (the other one) recommends "bit"; IEEE recommended "b"; ulgh!
